a10_octavia/api/drivers/driver.py

```diff
@@ -149,6 +152,43 @@ def member_update(self, old_member, new_member):
                    constants.MEMBER_UPDATES: member_dict}
         self.client.cast({}, 'update_member', **payload)
 
+    def member_batch_update(self, members):
```

Comment:
This looks like the old definition of the batch update  (https://github.com/openstack/octavia/blob/bd01ad94dcea3eca7636f1286ceed0753b563a28/specs/version1.1/enable-provider-driver.rst)

where as the one for victoria is defined in (https://github.com/openstack/octavia/blob/stable/victoria/doc/source/contributor/guides/providers.rst#batch-update) and https://github.com/openstack/octavia/blob/stable/victoria/octavia/api/drivers/amphora_driver/v1/driver.py#L255

Reply:
As this PR is meant for release v1.3.2 and as v1.3.2 is for stable/stein version, we have implemented the member_batch_update API similar as that of octavia's stable/stein member_batch_update API for now which will get merged into the release/v1.3.2 branch.

I think for victoria release, we'll need the implementation like https://github.com/openstack/octavia/blob/stable/victoria/octavia/api/drivers/amphora_driver/v1/driver.py#L255 which will get merged into our current master branch and not in release/v1.3.2 branch